<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiQtSocketClass</class>
 <widget class="QMainWindow" name="PiQtSocketClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PiQtSocket</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>客户端1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>客户端2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>280</width>
          <height>273</height>
         </rect>
        </property>
        <widget class="QLabel" name="client1Label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>261</height>
          </rect>
         </property>
         <property name="text">
          <string>老师好</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>279</width>
          <height>273</height>
         </rect>
        </property>
        <widget class="QLabel" name="client2Label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>271</width>
           <height>261</height>
          </rect>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="text">
          <string>我叫莫倪</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="clientText"/>
       </item>
       <item>
        <widget class="QPushButton" name="clientSend">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="client2Text"/>
       </item>
       <item>
        <widget class="QPushButton" name="client2Send">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PiQtSocket.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clientSend</sender>
   <signal>clicked()</signal>
   <receiver>PiQtSocketClass</receiver>
   <slot>on_client_send_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>client2Send</sender>
   <signal>clicked()</signal>
   <receiver>PiQtSocketClass</receiver>
   <slot>on_client2_send_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_client_send_clicked()</slot>
  <slot>on_client2_send_clicked()</slot>
 </slots>
</ui>
